docs/Users/Install.rst

```diff
@@ -6,12 +6,21 @@ platforms are Linux and Windows. *coala* is known to work on OS X as well.
 *coala* is tested against CPython 3.3, 3.4 and 3.5.
 
 In order to run *coala* you need to install Python. It is recommended,
-that you install Python3 >= 3.3 from http://www.python.org.
+that you install Python3 >= 3.3 from `here <https://www.python.org/downloads/>`_.
 
 The easiest way to install *coala* is using pip (Pip Installs Packages).
-If you don't already have pip, you can install it like described on
-https://pip.pypa.io/en/stable/installing.html. Note that pip is shipped
-with recent python versions by default.
+If you don't already have pip, you can install it like described in the
+`pip installation guide <https://pip.pypa.io/en/stable/installing.html>`_. Note
+that pip is shipped with recent python versions by default.
+
+Check if you have pip already installed and if so which version is installed,
+Type:
```

Comment:
this phrase is not correctly gramatically stated, and also the ``type`` shouldnt have a capital ``T``. rephrase it please. you can just leave it to: ``To check whether you have pip installed, type the following command which will also show you more information about your current pip version:``